tools/mosdepth.wdl

```diff
@@ -1,13 +1,57 @@
 ## # Mosdepth
 ##
-## This WDL tool wraps the [mosdepth tool](https://github.com/brentp/mosdepth).
+## This WDL file wraps the [mosdepth tool](https://github.com/brentp/mosdepth).
+## It also creates reference BEDs for restricting mosdepth coverage analysis to only exonic or coding regions.
 
 version 1.0
 
+task make_coverage_regions_beds {
```

Comment:
It feels a little odd to have this in the `mosdepth` tool. I don't have a better vision for it, other than potentially in `util`. It also uses a different docker image than `mosdepth`.

Reply:
Yea, maybe the best place for it is `util`, but that's getting awfully long. I can move it there for now, but I think part of my `code-cleanup` branch will be splitting `util` into some smaller files. Don't have an exact vision for it yet, but I'd agree it belongs in `util` for now.